testing/dart/image_filter_test.dart

```diff
@@ -18,6 +19,34 @@ const int greenCenterScaled = 0xFF00AA00;
 const int greenSideScaled = 0x80005500;
 const int greenCornerScaled = 0x40002B00;
 
+const List<double> greyscaleColorMatrix = <double>[
+  0.2126, 0.7152, 0.0722, 0, 0, //
+  0.2126, 0.7152, 0.0722, 0, 0, //
+  0.2126, 0.7152, 0.0722, 0, 0, //
+  0,      0,      0,      1, 0, //
```

Comment:
What's the `//` for? And why is it not used on other matrices below?

Reply:
I believe that the comments keep the formatter from wanting to compact the lines to the shortest. If there is a comment at the end of the line then it allows extra spaces. It would help on the matrix below that has a few 0.5's sprinkled in it, to keep the matrix elements lined up vertically...